contentcuration/contentcuration/models.py

```diff
@@ -377,29 +378,43 @@ class ContentNode(MPTTModel, models.Model):
     objects = TreeManager()
 
     def get_original_node(self):
-
+        key = "original_channel_{}".format(self.original_source_node_id or self.pk)
```

Comment:
original_source_node_id should match the target node's node_id, which gets preserved on ricecooker reruns